.circleci/config.yml

```diff
@@ -175,3 +175,16 @@ workflows:
           name: Test iOS on Device
       - android-device-checks:
           name: Test Android on Device
+
+  ui-tests-full-scheduled:
+    jobs:
+      - ios-device-checks:
+          name: Test iOS on Device - Scheduled
+      - android-device-checks:
+          name: Test iOS on Device - Scheduled
```

Comment:
I guess this should be named "Test Android on Device - Scheduled".

Reply:
🤦‍♂️Thanks! Fixed.